src/css/App.css

```diff
@@ -1,4 +1,7 @@
 .App {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
```

Comment:
💪 Make vertical centering possible. This shouldn't effect the rest of the app layout.